src/pages/index.jsx

```diff
@@ -427,10 +432,94 @@ const QuickstartsPage = ({ data, location }) => {
             </Overlay>
           </div>
 
-          {isSearchInputEmpty && (
+          {isSelectCategory && (
```

Comment:
I think we can consolidate these lines to just one logical expression

```suggestion
          {isSelectCategory && isSearchInputEmpty && (...)}
```

Reply:
@josephgregoryii Updated the changes